### PR TITLE
doc: regroup power optimization pages

### PR DESCRIPTION
**Pull request by greg-fer**
Split the current page in several pages.
This should make adding content to that section easier. NCSDK-19096.

### CODE REVIEW

**Comment by greg-fer** [comment]
This is how the section looks in the preview: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-11456/nrf/app_dev/optimizing/power.html

This PR is only meant to reorganize content as requested by PMT and get the section ready for further additions per protocol.

File: doc/nrf/app_dev/optimizing/power_general.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+   Current measurement on the Blinky sample with serial logging disabled
+
+The average current reduces to 6 µA, which implies 9.5 years of battery life on a 500 mAh lithium polymer battery compared to the 6-week battery life of the previous measurement.
+
+For a similar configuration, see the :ref:`udp` sample, which transmits UDP packets to an LTE network using an nRF9160 DK.
+You can use the sample to characterize the current consumption of the nRF9160 SiP.
+It is optimized for low power operation on the ``nrf9160dk_nrf9160_ns`` build target without any modifications.
+
+Idle current due to other peripherals
```
Comment (greg-fer) [inline doc/nrf/app_dev/optimizing/power_general.rst:72]:
```suggestion
Verify idle current due to other peripherals
```
To keep the style consistent.

**Review by jhn-nordic — APPROVED** [review]
Reorg looks good!

**Review by divipillai — APPROVED** [review]

**Review by maxd-nordic — APPROVED** [review]
lgtm

**Pull request merged**